tests/suites/test_suite_psa_crypto_slot_management.function

```diff
@@ -307,6 +328,10 @@ void create_existent( int lifetime_arg, int id_arg,
     size_t reexported_length;
     reopen_policy_t reopen_policy = reopen_policy_arg;
 
+    #if !defined(MBEDTLS_PSA_CRYPTO_KEY_EXTENDED_ID)
```

Comment:
Nit:Usual complaint about this belonging at the beginning of the function.

Reply:
Usual reply.